src/networks/common/graphqlApi/index.ts

```diff
@@ -105,15 +106,33 @@ export function createApi(uri: string, networkId: NetworkID): NetworkApi {
   return {
     loadProposalVotes: async (
```

Comment:
I'm a bit confused, why is this query in networks folder?

Reply:
This query or all queries overall?

`networks` directory contains interfaces (both network APIs, contract calls, constants, etc.) for interacting with supported networks (right now `starknet` and `evm`) in unified format. Those have separate actions and constants (due to differences between those), but the network API is shared and identical on both, so we have `common` directory there for things that are common between multiple networks, otherwise they would be in `network/starknet/api` or something like this. I think it makes sense to store those queries in networks directory, because if we had single network, or there was no common APIs between multiple networks those queries would still live in `networks` directory.

Reply:
Makes sense thanks.